color_sortting:
  offset:
  - -0.012000000000000007
  - 0.01
  - 0.002
  scale:
  - 1.02
  - 1.15
  - 1.0
  target_1:
  - 0.054
  - 0.243
  - 0.03
  target_2:
  - -0.016
  - 0.241
  - 0.03
  target_3:
  - -0.089
  - 0.237
  - 0.03
tag_sortting:
  offset:
  - -0.007
  - 0.007
  - 0.002
  scale:
  - 1.02
  - 1.11
  - 1.0
  target_1:
  - -0.085
  - 0.164
  - 0.016
  target_2:
  - -0.015
  - 0.168
  - 0.016
  target_3:
  - 0.056
  - 0.164
  - 0.016
tag_stackup:
  offset:
  - -0.007
  - 0.007
  - 0.002
  scale:
  - 1.02
  - 1.11
  - 1.0
  target_1:
  - -0.015
  - 0.164
  - 0.016
waste_classification:
  offset:
  - -0.01800000000000001
  - 0.011
  - -0.002
  scale:
  - 1.0
  - 1.11
  - 1.0
  target_1:
  - 0.093
  - -0.235
  - 0.025
  target_2:
  - 0.034
  - -0.239
  - 0.025
  target_3:
  - -0.032
  - -0.238
  - 0.025
  target_4:
  - -0.096
  - -0.235
  - 0.025
